src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -410,6 +407,67 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
         }
 
         internal static async Task<JObject> GetHostJsonExtensionsAsync(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, ILogger logger)
```

Comment:
This logic is specific to Durable Functions, right? Can we put Durable Functions in the method name to make this clear?

Reply:
`GetHostJsonExtensionsForDurableAsync` might be the one? @cgillum

Reply:
Sure

Reply:
Done